### PR TITLE
[Profile Viewer] Use Taming Levels Added to API. Maybe fix wardrobe + inv.

### PR DESCRIPTION
**Pull request by BigloBot**
Potential fix for active armour inserted into wrong wardrobe slot

### CODE REVIEW

File: src/main/java/de/hysky/skyblocker/skyblock/profileviewer/inventory/itemLoaders/WardrobeInventoryItemLoader.java [hunk 1]
```diff
@@ -25,9 +25,9 @@ public List<ItemStack> loadItems(JsonObject data) {
             if (activeSlot != -1) {
                 List<ItemStack> activeArmour = super.loadItems(activeArmorSet).reversed();
                 for (int i = 0; i < 4; i++) {
-                    int baseIndex = activeSlot % 9;
-                    int page = activeSlot / 9;
-                    int slotIndex = (page * 36) + (i * 9) + baseIndex - 1;
+                    int baseIndex = (activeSlot -1) % 9;
```
Comment (kevinthegreat1) [inline src/main/java/de/hysky/skyblocker/skyblock/profileviewer/inventory/itemLoaders/WardrobeInventoryItemLoader.java:28]:
```suggestion
                    int baseIndex = (activeSlot - 1) % 9;
```

**Review by kevinthegreat1 — DISMISSED** [review]
Looks fine. You can fix the formatting if you want.

**Review by kevinthegreat1 — APPROVED** [review]
Lgtm. But you'll have to fight it out with #874.

**Review by viciscat — APPROVED** [review]
i assume biglo tested this

**Pull request merged**